charts/seed-monitoring/charts/grafana/dashboards/owners/node-pool-dashboard.json

```diff
@@ -553,51 +553,51 @@
       ],
       "targets": [
         {
-          "expr": "sum(rate(node_cpu_seconds_total{mode!=\"idle\"}[$__rate_interval])) by (node) + on (node) group_right sum(kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}) by (label_worker_gardener_cloud_pool, node) - 1",
+          "expr": "  sum by (node) (rate(node_cpu_seconds_total{mode!~\"idle|iowait|steal\"}[$__rate_interval]))\n+ on (node) group_right ()\n  sum by (label_worker_gardener_cloud_pool, node) (\n    0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n  )",
```

Comment:
Do we need the additional whitespace at the beginning of the expression (ie. is part of the fancy formatting or can it be removed)?

Reply:
The leading whitespace is part of the formatting.
This is the new Prometheus formatting feature: https://github.com/prometheus/prometheus/pull/11039

<img width="600" alt="image" src="https://user-images.githubusercontent.com/23032437/187432493-4cf8dad0-4183-4ab4-a5bf-454339667f1e.png">
<img width="600" alt="image" src="https://user-images.githubusercontent.com/23032437/187432819-845b5801-e12c-4a5f-9073-bc4c9c889ebe.png">